Fixes PHPCS Errors based on WPCS

kopepasah opened the pull request:
This covers a first stab at fixing some of the PHPCS errors, which will enable us to test next iterations of this plugin more easily. 

Currently there is one final error to fix and it is blocked by #48.

Fixes #33

kopepasah commented:
Please do note that, at the creation of this PR, I have not tested these changes functionally. That is the next step.

danielbachhuber commented on pantheon-hud.php:
```diff
@@ -8,35 +8,38 @@
+		$last    = array_pop( $parts ); // File should be 'class-[...].php'.
+		$last    = 'class-' . $last . '.php';
+		$parts[] = $last;
+		$file    = dirname( __FILE__ ) . '/inc/' . str_replace( '_', '-', strtolower( implode( $parts, '/' ) ) );
```
```suggestion
		$file    = dirname( __FILE__ ) . '/inc/' . str_replace( '_', '-', strtolower( implode( '/', $parts ) ) );
```

danielbachhuber commented on phpcs.xml.dist:
```diff
@@ -1,4 +1,11 @@
+
+	<file>.</file>
+	<exclude-pattern>/vendor/</exclude-pattern>
 </ruleset>
```
Can we use a derivative of https://github.com/pantheon-systems/wp-saml-auth/blob/master/phpcs.xml.dist ? It has a few more dodads that are useful.

kopepasah replied:
Yeah, sound good. I can do this tomorrow (Tuesday).

kopepasah commented:
@danielbachhuber I updated the PHPCS config with the config file you reference and removed the unnecessary configuration. 

I did not change the PHP or WP version compatibility requirement configuration, as I am not of those without further digging. Hoping @ataylorme or yourself will know those minimum requirements. Either way, these need to be updated.

Let me know if there is something else missing.

danielbachhuber commented on pantheon-hud.php:
```diff
@@ -1,42 +1,45 @@
 /**
  * Plugin Name: Pantheon HUD
- * Version: 0.2.2
+ * Version: 0.2.1
```
```suggestion
 * Version: 0.2.2
```

danielbachhuber commented:
Looks great, thanks for your work on this!
